packages/flame/lib/src/camera/viewport.dart

```diff
@@ -66,6 +66,9 @@ abstract class Viewport extends Component
       camera.viewfinder.onViewportResize();
     }
     onViewportResize();
+    if (hasChildren) {
```

Comment:
Does this check optimize anything? The foreach should also just do a check if it is empty right?